requirements/requirements_doc.txt

```diff
@@ -4,3 +4,8 @@ sphinx-copybutton==0.5.0
 sphinx_toolbox==3.2.0
 sphinx_design==0.3.0
 sphinxcontrib-mermaid>=0.7.1
+sphinx-gallery==0.11.1
+Pillow>=9.0
+matplotlib==3.5.3
```

Comment:
It does look that the example does not involve `matplotlib` at all. Is this dependency required?